### DISCUSSION TITLE
Welcome to Algo-Tree Discussions!

### DISCUSSION THREAD

**Discussion by vatsalkesarwani12** [Algo-Phantoms/Algo-Tree#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ayu222220** [comment]
Hey!! My name  is Ayushi Singh  and I am 1st year B.tech student.I have been selected in GSSOC'21 and there I saw your project and did read everything about your and it really interested me. Would like to work with you. And most importantly I have very little knowledge of things but would love to explore. I am sorry if this is informal proposal but new to this so not exactly aware how to make proposal.

**Comment by maanpatel2492** [comment]
Hi,
My Name is Maan Patel.I have solved few of issue from Algo-tree project.Issues are not assigned to me.Still can I make a pull request?

**Reply by vatsalkesarwani12** [reply]
No, You can raise a valid Pr only for assigned issues.

**Comment by XZANATOL** [comment]
Want to make sure of followed timeline. Are pull request going to be reviewed after the 8th of march?
#GSSOC21

**Reply by vatsalkesarwani12** [reply]
Yes

**Comment by ghost** [comment]
Hello my name is Md Sadique Inam and I don't know how to contribute on Github and I saw this project of yours which looks very interested to me and I will be glad to contribute, 
how the issues are assigned to someone?
and how will I able to contribute?

**Reply by vatsalkesarwani12** [reply]
Read the readme

**Comment by BeAgarwal** [comment]
Can we start in Golang? It will be good if we have Golang language too.

**Reply by vatsalkesarwani12** [reply]
no not allowed
